### ISSUE TITLE
Bug in vinecop class

### ISSUE THREAD

**Issue by tvatter** [vinecopulib/rvinecopulib#129]
```
# specify pair-copulas
bicop <- bicop_dist("bb1", 90, c(3, 2))
pcs <- list(
    list(bicop, bicop),  # pair-copulas in first tree 
    list(bicop)          # pair-copulas in second tree 
)

# specify R-vine matrix
mat <- matrix(c(1, 2, 3, 1, 2, 0, 1, 0, 0), 3, 3) 

# set up vine copula model
vc <- vinecop_dist(pcs, mat)

# simulate some data
u <- rvinecop(50, vc)

# estimate a vine copula model
fit <- vinecop(u, "par", trunc_lvl = 1)

# simulate from truncated model
rvinecop(50, fit)

# Error in vinecop_inverse_rosenblatt_cpp(U, vinecop) : 
#    size of pair_copulas[0] does not match dimension of matrix (3); expected size: 1, actual size: 2 
```

**Comment by tnagler** [comment]
ok, this needs to be fixed before the next release

**Comment by tvatter** [comment]
Solved by #136
